packages/esm-appointments-app/src/tabs/appointments-table.scss

```diff
@@ -91,7 +89,8 @@
     background-color: $ui-01;
   }
 
-  :global(.bx--toolbar-content), :global(.bx--toolbar-search-container-expandable) {
+  :global(.bx--toolbar-content),
+  :global(.bx--toolbar-search-container-expandable) {
```

Comment:
We should avoid applying this global styles to carbon. It will after this styles as used on other components